region/brinstar/blue/Blue Brinstar Energy Tank Room.json

```
@@ -507,6 +507,34 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_ZebesIsAwake",
+        {"or": [
+          "h_CrystalFlash",
+          {"and":[
+            {"disableEquipment": "ETank"},
+            {"resourceAvailable": [{"type": "Energy", "count": 95}]},
```

```suggestion
            {"resourceAvailable": [{"type": "Energy", "count": 90}]},
```
Seems like there's no reason not to kill both Geemers, which allows us to slightly relax the energy requirement here.